### PR TITLE
Add support for channels

### PR DESCRIPTION
**Pull request by amberroy**
Issue #7.  Channel can be specified as a component parameter or in the URL.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,25 +20,35 @@ AFRAME.registerSystem('firebase', {
 
     if (!config) { return; }
 
+    this.channel = 'default';
```
Comment (ngokevin) [inline index.js]: Could do `this.channel = config.channel || 'default'`

File: index.js [hunk 2]
```diff
@@ -110,7 +121,7 @@ AFRAME.registerSystem('firebase', {
     var self = this;
     Object.keys(this.broadcastingEntities).forEach(function (id) {
       delete self.broadcastingEntities[id];
-      firebase.database().ref('entities/' + id).remove();
+      this.database.child('entities/' + id).remove();
```
Comment (ngokevin) [inline index.js]: `self.database`

File: index.js [hunk 3]
```diff
@@ -19,25 +20,35 @@ AFRAME.registerSystem('firebase', {
 
     if (!config) { return; }
 
+    this.channel = 'default';
+    if (config.channel) {
+      this.channel = config.channel;
+    }
+    var url = parse(location.href, true);
```
Comment (ngokevin) [inline index.js]: Can do all the parsing at the top of the module, outside the system, so we can just do:

```this.channel = config.channel || channelQueryParam || 'default'```

File: index.js (hunk 1 above)
Comment (amberroy) [inline index.js]: done.

File: index.js (hunk 2 above)
Comment (amberroy) [inline index.js]: done.

File: index.js (hunk 3 above)
Comment (amberroy) [inline index.js]: done.

**Pull request merged**